satellite/console/consoleweb/server.go

```diff
@@ -242,16 +241,14 @@ func (server *Server) bucketUsageReportHandler(w http.ResponseWriter, r *http.Re
 		zap.Stringer("since", since),
 		zap.Stringer("before", before))
 
-	ctx = console.WithAuth(ctx, auth)
-	bucketRollups, err := server.service.GetBucketUsageRollups(ctx, *projectID, since, before)
+	bucketRollups, err = server.service.GetBucketUsageRollups(ctx, *projectID, since, before)
 	if err != nil {
+		server.log.Error("bucket usage report error", zap.Error(err))
```

Comment:
I doubt that any error returned by `GetBucketUsageRollups` will be a `StatusUnauthorized`.

Reply:
It could be

Reply:
And in `defer` it was `StatusNotFound`, and a note that right now we have only error page

Reply:
I know that there is one page, but I think that the logic of distinguishing errors should be implemented and returning now just the `NotFound` because there is only one page.